pkg/inject/oauth_proxy_test.go

```diff
@@ -213,6 +289,40 @@ func TestOAuthProxyResourceLimits(t *testing.T) {
 	assert.Equal(t, *resource.NewQuantity(512, resource.DecimalSI), dep.Spec.Template.Spec.Containers[1].Resources.Requests[corev1.ResourceRequestsEphemeralStorage])
 }
 
+func TestOpenShiftOAuthProxyResourceLimits(t *testing.T) {
```

Comment:
If this is exactly like the previous test, consider making it a table test, so that it's clear what's different between the tests.